src/server_manager/cloud/gcp_api.ts

```diff
@@ -43,6 +43,62 @@ export type Instance = Readonly<{
   }>;
 }>;
 
+const GCE_V1_API = 'https://compute.googleapis.com/compute/v1';
+
+function projectLink(projectId: string): string {
```

Comment:
I think Url is a more appropriate name for these functions, and it aligns with the [API documentation](https://cloud.google.com/compute/docs/reference/rest/v1/instances/get#response-body) (see `zone` field of instance, for example).
```suggestion
function projectUrl(projectId: string): string {
```

Reply:
Done